### DISCUSSION TITLE
Bienvenido al foro de debate de La Palma DAO

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/lapalma#1] [General]
## 👋 ¡Bienvenido!
   Usamos este foro de debate como un lugar para conectarnos con otros miembros de nuestra comunidad. Esperamos que tu:
   * Te ciñas a los temas que tratamos aquí.
   * Compartas ideas.
   * Interactúes con otros miembros de la comunidad.
   * Des la bienvenida a los demás y tengas la mente bien abierta. Recuerda que esta es una comunidad que
   construimos juntos 💪.

  Para comenzar, haz una presentación de ti mismo y cuéntanos cómo te gustaría participar en esta comunidad.

**Comment by IBaixauli** [comment]
¿Qué te parece este foro de debate?

**Comment by ruth-ga** [comment]
Me parece interesante y necesario, para todos aquellos que deseamos y necesitamos intercambiar información y experiencias que redunden en ampliar nuestros conocimientos sobre criptomonedas , DAOs e informática en general.

**Comment by ruth-ga** [comment]
Uno de los temas principales retos  es encontrar medios y caminos para conseguir el ingreso de l Matics en nuestra billetera de METAMASK, de manera más económica y viable.
